web-client/src/views/JudgeActivityReport/JudgeActivityReport.tsx

```diff
@@ -223,6 +221,7 @@ export const JudgeActivityReport = connect(
               <div className="grid-col-auto display-flex flex-align-end">
                 <Button
                   className="position-relative margin-bottom-35"
+                  disabled={judgeActivityReportHelper.isFormPristine}
```

Comment:
FYI: The cursor won't change when hovering over this (disabled) button after https://github.com/ustaxcourt/ef-cms/pull/3470